.github/workflows/labeler.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@ac9175f8a1f3625fd0d4fb234536d26811351594 # v4.3.0
+    - uses: actions/labeler@8558fd74291d67161a8a78ce36a881fa63b766a9 # v5.0.0
```

Comment:
@Dhaulagiri this requires changes in our setup [it seems](https://github.com/hashicorp/design-system/actions/runs/7813047503/job/21311381094)

Reply:
🤕 oops. I'll take a peek and get it fixed